db/migrate/20150328144811_consolidate_plant_line_details.rb

```diff
@@ -0,0 +1,195 @@
+class ConsolidatePlantLineDetails < ActiveRecord::Migration
+  def up
+
+    # Rename existing PV to avoid naming clash
+    ActiveRecord::Base.connection.execute("ALTER TABLE plant_varieties \
+      RENAME TO old_plant_varieties")
+
+    create_table :plant_varieties do |t|
+      t.string :plant_variety_name,         primary: true
+      t.string :crop_type
+      t.string :comments
+      t.text :entered_by_whom,              default: "unspecified", null: false
+      t.date :date_entered
+      t.string :data_provenance
+
+      t.string :data_attribution,           default: "unspecified", null: false
+      t.string :country_of_origin,          default: "xxx", null: false
+      t.string :country_registered,         default: "xxx", null: false
```

Comment:
83 PVDs with country_of_origin not in countries.country_code. Most of them have multiple country codes in the field which calls for a m:n relationship. There are also three unidentified countries (XXX, CPB and PVD - @wjurkowski , can you explain these acronyms?) as well as a somewhat mysterious value of '<2'.

The `country_registered` field is a monumental mess which will require a smart curation task. I will get to work on the necessary transformations.